queries/typescript/textobjects.scm

```diff
@@ -1,32 +1,25 @@
 ; inherits: ecma
 
-
 ;; interface declaration as class textobject
-(interface_declaration) @class.outer
-
 (interface_declaration
-  body: _ @class.inner) @class.outer
+  body: (object_type)) @class.outer
```

Comment:
Can't you just remove the body, if you're not going to use `@class.inner`?

```lua
(interface_declaration) @class.outer
```

should work. Same for `type_alias_declaration` and `enum_declaration`.